wcivf/apps/people/templates/people/includes/_person_previous_elections_card.html

```diff
@@ -6,13 +6,16 @@
         <div class="ds-table">
             <table>
                 <caption style="font-style: normal;">
-                    <h3 class="ds-padded">{% trans "Previous Elections" %}</h3>
+                    <h3 class="ds-padded">{{ object.name }}'s {% trans "Previous Elections" %}</h3>
                 </caption>
                 <tr>
-                    <th>{% trans "Date" %}</th>
-                    <th>{% trans "Election" %}</th>
-                    <th>{% trans "Party" %}</th>
-                    <th>{% trans "Results" %}</th>
+                    <th>Date</th>
+                    <th>Election</th>
+                    <th>Party</th>
+                    <th>Results</th>
+                    {% if object.past_not_current_candidacies.first.previous_party_affiliations %}
```

Comment:
Missing the `exists` here.

Although looking at this now, the issue with displaying the details in this table is that it will only appear to the user _after_ the election has happened (when the candidacy becomes a past ballot). Which means it loses its usefulness somewhat? But perhaps this was already something discussed with @pmk01?